recipe/meta.yaml

```diff
@@ -305,3 +306,4 @@ extra:
     - dHannasch
     - h-vetinari
     - vnlitvinov
+    - krfricke
```

Comment:
welcome on board! 🙃

Reply:
Thanks 😄

Reply:
Agreed, already mentioned that on the last PR - happy to have you on board!